guidelines/REUSE.md

```diff
@@ -205,8 +232,11 @@ To include these parameters, you would need to add them individually as such:
 ```
 
 ### Operations
+
 Again, Operations can be shared across files.  Although the reusability of operations will be less than with Parameters and models. For this example, we will share a common `health` resource so that all APIs can reference it:
```

Comment:
Good point on consistency...I'm also trying to move to 'Definitions' to match the convention (we had model/schema/definition pretty inconsistently all over).